core/FamousEngine.js

```diff
@@ -351,9 +351,13 @@ FamousEngine.prototype.message = function message (command) {
  *
  * @return {Scene} a new instance of Scene.
  */
-FamousEngine.prototype.createScene = function createScene (selector) {
+FamousEngine.prototype.createScene = function createScene (selector, options) {
     selector = selector || 'body';
 
+    this.message('CONTEXT');
```

Comment:
This would mean we can get rid of `getOrSetContext` and replace it with a simple getter.